controllers/userController.ts

```diff
@@ -12,7 +12,7 @@ const host = process.env.HOST || "localhost";
  * @param {string} username - The username of the user
  * @param {string} email - The email address of the user
  * @param {string} password - The password of the user *encrypted
- * @param {Optional<Image>} profile - Profile display image
+ * @param {string} profile - Profile display image
```

Comment:
Kayla and I discussed in class that we'll use `string` for now since that's what the backend says. Kayla pointed out that it probably should be optional throughout though, since there is a default pfp.

Kayla, please create a follow up task to change frontend and backend to make this Optional